src/containers/navigation-icons/AccountIcon.tsx

```diff
@@ -0,0 +1,67 @@
+import { useCallback, FC, MouseEvent } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useAppSelector } from '~/hooks/use-redux'
+import { AxiosResponse } from 'axios'
+
+import Avatar from '@mui/material/Avatar'
+import Tooltip from '@mui/material/Tooltip'
+
+import { userService } from '~/services/user-service'
+import useAxios from '~/hooks/use-axios'
+import { defaultResponses } from '~/constants'
+
+import { styles } from '~/containers/navigation-icons/NavigationIcons.styles'
+
+import { UserResponse, UserRole } from '~/types'
+
+interface AccountIconProps {
+  openMenu: (event: MouseEvent) => void
+}
+
+const AccountIcon: FC<AccountIconProps> = ({ openMenu }) => {
+  const { t } = useTranslation()
+  const { userId, userRole } = useAppSelector((state) => state.appMain)
+
+  const getUserData: () => Promise<AxiosResponse<UserResponse>> = useCallback(
+    () => userService.getUserById(userId, userRole as UserRole),
+    [userId, userRole]
+  )
+
+  const {
+    loading,
+    response: { photo, firstName, lastName }
+  } = useAxios<UserResponse>({
+    service: getUserData,
+    fetchOnMount: true,
+    defaultResponse: defaultResponses.object as UserResponse
+  })
+
+  if (loading) {
+    return <Avatar sx={styles.accountIcon} />
+  }
+
+  let userPhoto = ''
```

Comment:
redundant variable, just pass `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}` into src

Reply:
It was made for increasing readability of code.
But I didn't consider that as redundant variable(((
thank you. Fixed

Reply:
You use it only in one place and it isn`t a very long statement, also, defining variables with let keyword isn`t a good practice at all